src/redux/modules/auth/__tests__/reducer.test.js

```diff
@@ -64,10 +90,10 @@ describe('(Reducer) auth', () => {
   });
 
   it('should handle LOGOUT_SUCCESS', () => {
-    expect(authReducer(undefined, { type: authTypes.LOGOUT_SUCCESS, error: 'fakeError' })).to.deep.equal({
+    expect(authReducer(undefined, { type: authTypes.LOGOUT_SUCCESS, isLoggingOut: 1234, error: 'fakeError' })).to.deep.equal({
       isAuthenticated: false,
       isFetching: false,
-      isLoggingOut: false,
+      isLoggingOut: 1234,
```

Comment:
why 1234?

Reply:
Lol, i was trying to be funny. but yes that def should be boolean. Ill fix that.